### ISSUE TITLE
Adding Royalty-Free LOTR inspired music

### ISSUE THREAD

**Issue by Shooteger** [Helliaca/HoI4-LotrMod#117]
Hello, I know a submod exists, but the silence robs of atmosphere. Maybe we can add some royality and copyright free music, a total of 1-3 hours in track which sound similar. (Many great free songs out there) Here an example: https://www.youtube.com/watch?v=bBCMZsAL4mU

Just as an idea, maybe its not neccasary, but I think it would add much atmosphere.

**Comment by Helliaca** [comment]
Yes, this is a good idea. The music from your linked video is great as well, but based on what's in the description it seems to come from a lot of sources and I'm not entirely sure they have permission to distribute them all as "royalty free".

Will have to look a bit more into it, but the idea is definitely something worth pursuing

**Comment by Shooteger** [comment]
> Yes, this is a good idea. The music from your linked video is great as well, but based on what's in the description it seems to come from a lot of sources and I'm not entirely sure they have permission to distribute them all as "royalty free".
> 
> Will have to look a bit more into it, but the idea is definitely something worth pursuing

Great! :) Yeah in that video I think 4 tracks are the originals... But there are tons out there without, better than just silence...
